DEVELOPMENT.md

```diff
@@ -10,57 +10,67 @@ Athens uses [Go Modules](https://golang.org/cmd/go/#hdr-Modules__module_versions
 
 # Services that Athens Needs
 
-Athens relies on several services (i.e. databases, etc...) to function properly. We use [Docker](http://docker.com/) images to configure and run those services. **However, Athens does not require any storage dependencies by default**. The default storage is in memory, you can opt-in to using the `fs` which would also require no dependencies. But if you'd like to test out Athens against a real storage backend (such as MongoDB, Minio, S3 etc), continue reading this section:
+By default, Athens doesn't require any external services to be running. By default, it uses in-memory storage and you can also opt into on-disk storage, and neither method requires any external services.
+
+However, Athens does support other storage drivers, and takes advantage of other external services that can be helpful for more permanent and "production-like" deployments. Here are some of the services that Athens can take advantage of:
+
+- [Minio]((https://minio.io/)) storage driver
+- [MongoDB](https://www.mongodb.com/) storage driver
+- [Jaeger](https://www.jaegertracing.io/) for distributed tracing
+
+## How Do I Run All These Services Locally?
+
+We use [Docker](https://docker.com) images and [Docker Compose](https://docs.docker.com/compose/) to start up Athens and all of those services, and connect them all together. Thanks to all that magic, you can start up all an Athens server alongside all its services with one command! :tada: :rocket:
 
 If you're not familiar with Docker, that's ok. In the spirit of Buffalo, we've tried to make
 it easy to get up and running:
 
-1. [Download and install docker-compose](https://docs.docker.com/compose/install/) (docker-compose is a tool for easily starting and stopping lots of services at once)
-2. Run `make dev` from the root of this repository
+1. [Download and install docker-compose](https://docs.docker.com/compose/install/)
+1. Run `make dev` from the root of this repository
+1. Access Athens on `http://localhost:3000`
 
-That's it! After the `make dev` command is done, everything will be up and running and you can move
-on to the next step.
+That's it! Athens will take a few seconds to configure and connect to all the services it uses. When it's done, everything will be up and running and ready to use.
 
 If you want to stop everything at any time, run `make down`.
 
-Note that `make dev` only runs the minimum amount of dependencies needed for things to work. If you'd like to run all the possible dependencies run `make alldeps` or directly the services available in the `docker-compose.yml` file. Keep in mind, though, that `make alldeps` does not start up Athens or Oympus, but **only** their dependencies.
+>`make down` will stop all the resources that `make dev` started. If you want to completely delete all the unused Docker resources on your system, run `docker system prune`. This command may delete other unused resources that you didn't create with `make dev`
```

Comment:
A bit nit picking but do we need the part about pruning. Won't `docker-compose down` tear down any resources and their networks? Thay we don't need to suggest people run `docker system prune` will `unused containers, networks, images (both dangling and unreferenced)` which could get rid of resources they need elsewhere.

Reply:
Yea, it does. The problem comes in if they're not in the athens directory, but to be honest that's mostly my problem and I let my extreme solution leak into the docs 😆 . I'll take this out.